release-content/0.14/release-notes/11658_Add_Rotation2d.md

```diff
@@ -1 +1,12 @@
-TODO
+Ever wanted to work with rotations in 2D and get frustrated with having to choose between quaternions and a raw `f32`?
+Us too!
+
+We've added a convenient [`Rotation2d`](https://dev-docs.bevyengine.org/bevy/math/struct.Rotation2d.html) type for you, with plenty of helper methods.
+Feel free to replace that helper type you wrote, and submit little PRs for any useful functionality we're missing.
+
+`Rotation2d` is a great complement to the [`Dir2`](https://dev-docs.bevyengine.org/bevy/math/struct.Dir2.html) type (formerly `Direction2d`).
+The former represents an angle, while the latter is a unit vector.
+These types can be readily transformed into each other, and the most convenient representation depends heavily on the task at hand.
```

Comment:
I don't think you can really *transform* between them, you can easily use Rotation2d to rotate Dir2.

Reply:
I mean, you can transform between them, and turning rotations into directions is a pretty common thing to want to do in 2d so I think calling it out directly is probably good.

Reply:
Hmm. That should be possible, unless I'm badly misunderstanding something. Lemme make a PR.

Reply:
[#13635](https://github.com/bevyengine/bevy/pull/13635) addresses this gap in the API.

Reply:
That pr stalled, and can't easily be improved because `angle_between` is exposed directly from the glam API. So how about the following:

```suggestion
These types are similar but not interchangeable, and the choice of representation depends heavily on the task at hand. You can rotate a direction using `direction = rotation * Dir2::X`. To recover the rotation, use `Rotation2d::radians(Dir2::X.angle_between(direction))`.
```